processes/ingest/doab.py

```diff
@@ -56,7 +56,7 @@ def runProcess(self):
             self.saveRecords()
             self.commitChanges()
 
-            logger.info(f'Ingested {len(self.records) if records else 1} DOAB records')
+            logger.info(f'Ingested {len(self.records) if records else 0} DOAB records')
```

Comment:
This doesn't quite work due to the singleRecord condition above. It should probably be `len(records) if records else (1 if record else 0)`

Reply:
Oh ok that makes sense. I appreciate the clarification.